examples/tutorials/03_fetch_data.py

```diff
@@ -0,0 +1,29 @@
+from iota import Iota, Address
+from iota.codecs import TrytesDecodeError
+
+# Declare an API object
+api = Iota('https://nodes.devnet.iota.org:443', testnet=True)
+
+# Address to fetch data from
+# Replace with your random generated address from Tutorial 2. to fetch the data
+# that you uploaded.
+addy = Address(b'WWO9DRAUDDSDSTTUPKJRNPSYLWAVQBBXISLKLTNDPVKOPMUERDUELLUPHNT9L9YWBDKOLYVWRAFRKIBLP')
+
+print('Fetching data from the Tangle...')
+# Fetch the transaction objects of the address from the Tangle
+response = api.find_transaction_objects(addresses=[addy])
+
+if not response['transactions']:
+    print('Couldn\'t find data for the given address.')
+else:
+    print('Found:')
+    # Iterate over the fetched transaction objects
+    for tx in response['transactions']:
+        # data is in the signature_message_fragment attribute as trytes, we need
+        # to decode it into a unicode string
+        try:
+            data = tx.signature_message_fragment.decode()
+        # Ignore invalid or not unicode encoded data 
+        except (TrytesDecodeError, UnicodeDecodeError):
+            continue
```

Comment:
I think we can achieve the same result with:

```python
data = tx.signature_message_fragment.decode(errors='ignore')
```

More info:
https://github.com/iotaledger/iota.py/blob/2.1.0/iota/types.py#L536-L547

(this is meant to mimic the way Python's [`codecs.decode()` function](https://docs.python.org/3/library/codecs.html#codecs.decode) works)

Reply:
You are right, however, with `ignore`, only the values that can't be decoded are dropped, the rest of the non-meaningful message will be printed.
Anyway, I added a small explanation and changed it to your suggestion.